docs/mediapipe.md

```diff
@@ -204,6 +204,37 @@ Graphs can also be configured in the `mediapipe_config_list` section.
 ```
 Nodes in the MediaPipe graphs can reference both the models configured in model_config_list section and in subconfigs.
 
+#### Simplified mediapipe graphs and LLM models loading
```

Comment:
it was already covered in Starting OpenVINO Model Server with Mediapipe servables